File: c/iniciante/1002.c
#include <stdio.h>

double return_area(double raio) {
    const double pi = 3.14159;

    return (pi * (raio * raio));
}

void print_area(double area) {
    printf("A=%.4f\n", area);
}

int main(void) {
    double raio;
    scanf("%lf", &raio);

    print_area(return_area(raio));
    return 0;
}

File: python/iniciante/1002.py
'''
title: Área do Círculo
number of the problem: 1002
date: 27/02/2018
author: ltsdw
'''

def area(raio: float) -> float:
    pi: float = 3.14159
    
    return pi*raio**2


raio = float(input(''))

print(f'A={area(raio):.4f}')
